### DISCUSSION TITLE
DJI Vids show GPS Data in Video

### DISCUSSION THREAD

**Discussion by Stapelgabler** [rocksdanister/lively#842] [General]
When I set videos of my DJI drone as background appear on half of the screen GPS and other data of the drone but this is not visible when I e.g. under the gallery this is not displayed. I have read on the Internet that you can use Dashware to do something like this9 but I am not very familiar with it. Thanks in advance

**Comment by Stapelgabler** [comment]
never used github before sry for mistakes

**Reply by rocksdanister** [reply]
I don't have DJI drone.. but can't you like get the recorded video files and set that as wallpaper?

**Comment by Stapelgabler** [comment]
Thanks for the quick help i'll try it out.
